spec/classes/mod/proxy_html_spec.rb

```diff
@@ -20,7 +20,7 @@
 
     context 'on i386' do
       let(:facts) do
-        super().merge(hardwaremodel: 'i686',
+        super().merge(os: { family: 'Debian', name: 'Debian', release: { full: '11', major: '11' }, hardware: 'i686', },
```

Comment:
I think it would be good to fix https://github.com/voxpupuli/rspec-puppet-facts/issues/112 and solve it at that level. It's already pulled in and it's probably the right place to fix it (or rspec-puppet itself). IMHO that would be best for the ecosystem. Fixing it in voxpupuli-test was just my way of making quick progress.

I can't promise to have time for it, but if someone contributes the PR I'd be happy.

Reply:
Was unable to include the voxpupuli-test gem added due to conflicts with existing gems so have simply added the override_facts code to the spec_helper_local.rb for now if you don't mind me copying your code.